modules/nw-external-dns-operator.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * networking/understanding-external-dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-operator_{context}"]
+= External DNS Operator
+
+The External DNS Operator implements the External DNS API from the `olm.openshift.io` API group. The External DNS Operator deploys the `ExternalDNS` using a deployment. The `external-dns-operator` deployment watches the resources such as services and routes in the cluster and updates the External DNS providers.
+
+.Procedure
+
+You can deploy the ExternalDNS Operator on demand from the OperatorHub with a `Deployment` object.
+
+. Use the `oc get` command to view the `Deployment` status:
++
+[source,terminal]
+----
+$ oc get -n external-dns-operator deployment/external-dns-operator
+NAME                    READY     UP-TO-DATE   AVAILABLE   AGE
+external-dns-operator   1/1       1            1           23h
+----
+
+. Use the `oc get` command to view the state of the External DNS Operator:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get subscription external-dns-operator -o yaml | yq e '.status'
```

Comment:
This command can be removed if we add the check of the `installplan` before.

Reply:
Ack'ed